fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -34,57 +44,55 @@ private FHIRRegistry() {
         providers = loadProviders();
```

Comment:
Unrelated to this PR, but when does FHIRRegistry get initialized?  I know Mark was asking for that to happen during server start, but wasn't sure if it was addressed in this PR or not.

Reply:
It is initialized in servlet context listener

Reply:
ah, now I see it in FHIRServletContextListener.java  ...looks good